.env.example

```diff
@@ -0,0 +1,12 @@
+# Local development environment variables for Docker Compose
+# Copy this file to .env and fill in your actual values
+#
+# Usage:
+#   cp .env.example .env
+#   # Edit .env with your actual secrets
+#   docker compose up
+
+# Organization API codes for waste-movement-backend service
+ORG_API_CODES=your-org-api-codes-here
+WASTE_TRACKING_ID_BACKEND_VERSION=latest
+WASTE_MOVEMENT_BACKEND_VERSION=latest
```

Comment:
I think this should be `WASTE_TRACKING_EXTERNAL_API_VERSION`, not `WASTE_MOVEMENT_BACKEND_VERSION`